yarn.lock

```diff
@@ -1452,7 +1452,7 @@
 
 "@moltin/sdk@./tmp/moltin-sdk-v0.0.0-semantic-release.tgz":
   version "0.0.0-semantic-release"
-  resolved "./tmp/moltin-sdk-v0.0.0-semantic-release.tgz#b8828ced23616cca5f80f731a2070e0c0696f75e"
+  resolved "./tmp/moltin-sdk-v0.0.0-semantic-release.tgz#0b5baa9e1d0942ff6a866e2ba7f7c6fd4aa28d9c"
```

Comment:
Prior to release, we'll have to revert this from the `external-auth-integration` branch and remove the bundled .tgz so it doesn't reach master.

Reply:
Yes